src/pages/Account/Profile.tsx

```diff
@@ -61,57 +65,98 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
     history.push(IS_BURN ? `/enter/step3` : nextUrl);
   };
 
+  const profilePrefilledData = getLocalStorageItem(
+    LocalStorageItem.prefillProfileData
+  );
```

Comment:
I believe we can read the provided data as claims off the firebase auth identity token always can't we? So do we even need to use `localStorage` like this?

Reply:
This is where I believe it would live. Not 100% though, and can't really test as I don't have SSO creds:

```
firebase
  .auth()
  .currentUser?.getIdTokenResult()
  .then(
    (token) => token.claims
  );
```

Reply:
See also: https://github.com/sparkletown/sparkle/pull/1667#discussion_r659472381

Reply:
Mappings paths for reference:
![image](https://user-images.githubusercontent.com/753891/123587871-43a45000-d82a-11eb-9ef5-0b07383646fe.png)

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/c77cd255fb597da0dd2611bc66089e77dbe0e59f

Reply:
Just to confirm, did you test this and see that it works? I was assuming, and couldn't test it since I don't have creds.

Reply:
@0xdevalias yes, I did test it